full_documentation.md

```diff
@@ -68,6 +74,18 @@ In some cases, you might just be working with a few files. If so, you can skip t
 ### Situation 8: Google's from AWS
 [Please see Google's documentation.](https://cloud.google.com/migrate/compute-engine/docs/4.8/how-to/migrate-aws-to-gcp/migrating-aws-vms)
 
+### Situation 9: Importing from Gen3 (an NIH cloud-platform partner)
+[Gen3](https://gen3.org/) is an NIH cloud-platform that currently interoperates with Terra as the data provider for [NHLBI BioData Catalyst](gen3.biodatacatalyst.nhlbi.gov) and will eventually support NHGRI AnVIL in a similar role. Gen3 emp
```

Comment:
Looks like an unfinished sentence, can you clarify what you meant here?

Reply:
I don't remember what I was planning to say here... maybe just delete "Gen3 emp" for now :)